Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/BoundingBox/BoundingBox.cs

```diff
@@ -2181,6 +2176,7 @@ public void OnSourceDetected(SourceStateEventData eventData)
                 if (sourcesDetected.Count == 0 || sourcesDetected.Contains(eventData.Controller) == false)
                 {
                     sourcesDetected.Add(eventData.Controller);
+                    Debug.Log(eventData.Controller.InputSource.SourceName);
```

Comment:
Might be good to remove this before checking in

Reply:
yup= removed.